src/mapport.cpp

```diff
@@ -192,21 +192,21 @@ static bool ProcessUpnp()
 
             if (r != UPNPCOMMAND_SUCCESS) {
                 ret = false;
-                LogPrintf("AddPortMapping(%s, %s, %s) failed with code %d (%s)\n", port, port, lanaddr, r, strupnperror(r));
+                LogPrintLevel(BCLog::NET, BCLog::Level::Warning, "UPnP: AddPortMapping(%s, %s, %s) failed with code %d (%s)\n", port, port, lanaddr, r, strupnperror(r));
```

Comment:
was the message changed on purpose to confirm to the rest of the logs?
Shouldn't we rather remove the prefixes now that the category is specified?

Reply:
Yes, this was on purpose. i've kept the prefix because nat-pmp and UPnP messages should be distinguishable.